src/h3lib/lib/algos.c

```diff
@@ -411,6 +411,32 @@ H3Index h3NeighborRotations(H3Index origin, Direction dir, int* rotations) {
     return out;
 }
 
+/**
+ * Get the direction for a given neighbor. This is effectively the reverse
```

Comment:
This gives the direction to move from origin to destination?
Please add a comment about what this does in error cases

Reply:
Will update comment

Reply:
Did you want more error explanation than `Returns INVALID_DIGIT if the cells are not neighbors`? This is the only error case AFAIK, though invalid indexes might incorrectly return a direction (depending on what `h3NeighborRotations` does with invalid indexes).